doc/development/deprecations.rst

```diff
@@ -64,6 +78,12 @@ Completed deprecation cycles
   - Deprecated in v0.33
   - Removed in v0.34
 
+* The ``prep`` keyword argument in ``QuantumScript`` has been removed.
```

Comment:
merge issue?

Reply:
fixing an old merge issue! during the `prep` deprecation, a bad merge dropped it way down the list, but it should be near the top because it's introduced in v0.33. I didn't notice at first, but @Alex-Preciado caught it for me!